pkg/btputils/structures.go

```diff
@@ -0,0 +1,118 @@
+package btputils
```

Comment:
Looks good 👍. One small thing: I would move structures.go to the btp package.I think it does not make sense to create a whole package just for some structures